api/services/roundService.js

```diff
@@ -0,0 +1,30 @@
+
+export default class RoundService {
+    constructor(players) {
+        this.players = players;
+
+    }
+
+    newRound() {
+        getCzar(newGame)//TODO: newgame is a boolean that is passed in
+
+    }
+
+    endRound() {
+
+    }
+
+    getCzar(newGame) {
+        if (newGame) {
+            this.currentCzarNum = Math.floor(Math.random() * this.players.length -1 );
+        } else {
+            if (this.currentCzarNum === this.players.length) {
```

Comment:
Do you this would be neater with a ternary statement? I'm not sure either way..
`this.currentCzarNum = (this.currentCzarNum === this.players.length) ? 0: this.currentCzarNum + 1;`
replacing lines 21 - 25